challenge-07/challenge-07.js

```diff
@@ -54,7 +64,13 @@ Só vamos mudar o range:
 - No segundo "for", mostre os números ímpares entre 111 e 125, inclusive eles.
 */
 console.log( 'Números pares entre 100 e 120:' );
-// ?
+for(var num = 100; num <= 120; num++) {
+	num % 2 === 0 ? console.log(num) : '';
```

Comment:
-1
Procure usar ternários somente em dois casos: 
1. Quando for atribuir o valor resultante do ternário à uma variável;
2. Quando for retornar o resultado de uma função.

De outra forma, use o `if` mesmo, ok? :)